xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1030 @@
+      mirror.
+     </para>
+     <para>
+      If the cluster is deployed before the mirror is running, you must
+      reconfigure all nodes manually.
+     </para>
+    </note>
+    <para>
+     Set up &rmtool; on the two mirror servers.
+    </para>
+    <para>
+     <link xlink:href="https://www.suse.com/documentation/sles-15/book_rmt/data/cha_rmt_installation.html">Follow these instructions</link>
+    </para>
+    <procedure>
+     <title>Configure The External Mirror</title>
+     <step>
+     <para>
+      Connect the external mirror to &scc; as described in <link xlink:href="https://www.suse.com/documentation/sles-15/book_rmt/data/sec_rmt_mirroring_credentials.html">these instructions</link>.
+     </para>
+    </step>
+    </procedure>
+
+    <procedure>
+     <title>Configure The Internal Mirror</title>
+     <step>
+      <para>
+       You need to disable the automatic repository sync on the internal server.
+       This will attempt to download information from &scc; which can not be
+       reached from inside the airgapped network.
+      </para>
+      <screen>&prompt.root;systemctl disable rmt-server-sync.timer
+      </screen>
+     </step>
+    </procedure>
+    <para>
+     Now you need to perform the update procedure to do an initial sync of data
+     between the upstream sources and the external mirror and the external and
+     internal mirrors. Refer to: <xref linkend="sec.deploy.scenarios.airgap.rpm-repository.update"/>.
+    </para>
+   </sect3>
+
+   <sect3 xml:id="sec.deploy.scenarios.airgap.rpm-repository.client">
+    <title>Client Configuration</title>
+    <para>
+     <link xlink:href="https://www.suse.com/documentation/sles-15/book_rmt/data/cha_rmt_client.html">Follow these instructions</link>
+     to configure all CaaSP nodes to use the package repository mirror server in
+     the airgapped network.
+    </para>
+    <para>
+
+    </para>
+   </sect3>
+   <sect3 xml:id="sec.deploy.scenarios.airgap.rpm-repository.update">
+    <title>Updating The Mirror</title>
+    <para>
+     <link xlink:href="https://www.suse.com/documentation/sles-15/book_rmt/data/sec_rmt_mirroring_export_import.html">Follow these instructions</link>
+     to update the external server, transfer the data to a storage device, and
+     use that device to update the airgapped server.
+    </para>
+   </sect3>
+  </sect2>
+
+  <sect2 xml:id="sec.deploy.scenarios.airgap.container-registry">
+   <title>Container Registry Mirror</title>
+   <note>
+    <title>Mirroring Multiple Image Registries / Chart Repositories</title>
+    <para>
+     You can mirror images and charts from multiple registries in one shared
+     internal registry. Do note, however, that you must manually resolve
+     conflicts due to duplicate charts and container images.
+    </para>
+    <para>
+     We highly recommend running separate helm chart and container registry
+     mirrors for each source registry.
```

Comment:
Charts from somewhere else would mean also mirroring the referenced images, this is currently not possible on the same machine because the container expects the config in exactly one location that is already taken by our SUSE specific registry

Reply:
I'm fine with this wording. Marking as resolved.